tox.ini

```diff
@@ -35,7 +35,7 @@ extras =
 commands_pre=
     numpydev: pip install -U --pre -i https://pypi.anaconda.org/scientific-python-nightly-wheels/simple numpy
     devdeps: pip install -U --pre -i https://pypi.anaconda.org/liberfa/simple pyerfa
-    devdeps: pip install -U --pre -i https://pypi.anaconda.org/astropy/simple astropy
+    devdeps: pip install -U --pre -i git+https://github.com/lpsinger/astropy@masked-classes-importable
```

Comment:
```suggestion
    devdeps: pip install git+https://github.com/lpsinger/astropy@masked-classes-importable
```

Reply:
Oh right, your fork won't have the right tags for setuptools_scm. You can make a fake tag on your fork, or otherwise relax any astropy pins here. The former might be easier.